app/views/admin/projects.html.erb

```diff
@@ -73,52 +48,107 @@ See doc/COPYRIGHT.rdoc for more details.
           <label for='name'><%= Project.model_name.human %>:</label>
           <%= text_field_tag 'name', params[:name] %>
       </li>
+      <li class="simple-filters--filter">
+        <label for='' aria-visible="false" style="visibility: hidden"><%= l(:button_apply) %></label>
```

Comment:
Okay, help me. What is the purpose of this label?

* it does not indicate to which element it belongs (the button)
* I can't even find `aria-visible` on the internet, do you mean `aria-hidden`? Also note [this article](http://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/) on the topic
* shouldn't the button already be self-explanatory (i.e. you only add the text here, that is already present on the button)?